### DISCUSSION TITLE
Playback controls aren't present when live spectating

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#26931] [Q&A]
I used it to speed up the play and catch up with the most recent frames, but it's useful for analysing too.
![image](https://github.com/ppy/osu/assets/53081750/0e393952-d80b-405f-8f48-7320404f4c0b)

2024.131.0-lazer

**Comment by Rekunan** [comment]
I imagine this could work similarly to the playback controls in YouTube livestreams, letting you rewind, seek, change speed, etc.

**Reply by Theighlin** [reply]
It's the same as it's everywhere else, it was already there not too long ago... Not sure if it was removed by mistake or there was a reason i missed.

**Reply by Rekunan** [reply]
That sounds like it should be an issue and not a discussion then. When was the last time it was there?

**Reply by Joehuu** [reply]
See https://github.com/ppy/osu/pull/26446#issuecomment-1893483318 for reasoning why it was removed in the PR before merge.
